### PR TITLE
Better code formating and some minor fixes and enhancments

### PR DESCRIPTION
**Pull request by moty66**
I know that I had to make small commits, but I started to write and I din't stop :)
anyway, here is a list of the changes I made

* Param parsing function, make things easier when adding new params in future
* Better output code
* Fixes for all php notices and warnings
* Better source code formatting and comments

### CODE REVIEW

File: ddos.php [hunk 1]
```diff
@@ -1,100 +1,216 @@
...
 }
 
-if(isset($_GET['host']) && (isset($_GET['time']) && is_numeric($_GET['time'])) || (isset($_GET['packet']) && is_numeric($_GET['packet']))){
+// Verify if params are correct
+if(!empty($params['host']) && (is_numeric($params['time'])) || is_numeric($params['packet'])){
 	
 	// If executed from CLI no password
-	if($cli==false){
+	if(!is_cli()){
 		if(isset($_GET['pass'])){
```
Comment (TheZ3ro) [inline ddos.php]: It's not $_GET['pass'] anymore, but it's $params['pass']

**Comment by TheZ3ro** [comment]
Lot of changes :)

I want to ask you a question,
Why have you placed the "exec_time==0" if outside the packet attack if?

If it's a time attack and I set time to 0, setting the attack IF like this:
"if($params['time']){" 
in case the time param is 0, the if is false so change automatically to packet attack.

I hope you understand what I'm saying :smile:

**Comment by moty66** [comment]
Opps, you are right, mea culpa

**Comment by TheZ3ro** [comment]
Also, on [L151](https://github.com/moty66/DDoS-PHP-Script/blob/master/ddos.php#L151) 
It's not $_GET['pass'] anymore, but it's $params['pass']

Am I wrong?

Anyways, I think that we can merge :+1:

**Comment by moty66** [comment]
fix in commit c8e69fd

**Comment by TheZ3ro** [comment]
I was late :dash:

**Pull request merged**